respecConfig.js

```diff
@@ -16,9 +15,7 @@ var respecConfig = {
     height: 100,
     id: 'spec-logo',
   }],
-  wg: "Web Platform Incubator Community Group",
-  // URI of the public WG page
-  wgURI: "https://www.w3.org/community/wicg/",
+  group: "wicg",
   github: "https://github.com/wicg/serial",
-  xref: ["STREAMS", "DOM", "WebIDL", "HTML"]
+  xref: ["DOM", "HTML", "PERMISSIONS-POLICY", "STREAMS", "WebIDL"]
```

Comment:
Currently Streams and Permissions Policy are non-normative references; they should probably be normative. (Or maybe this is happening because of my problem with `[= =]` cross references.)

Reply:
Yep, they got moved to normative once the xref database fixed itself.